client/settings/settings-warnings/incompatibility-notice.js

```diff
@@ -7,48 +7,35 @@ import interpolateComponents from '@automattic/interpolate-components';
 /**
  * Internal dependencies
  */
-import { Notice } from '@wordpress/components';
-import NoticeOutlineIcon from 'gridicons/dist/notice-outline';
 import './style.scss';
+import InlineNotice from 'wcpay/components/inline-notice';
 
 const IncompatibilityNotice = ( { message, learnMoreLinkHref } ) => (
-	<Notice
+	<InlineNotice
 		status="warning"
+		icon={ true }
 		isDismissible={ false }
-		className="express-checkout__notice express-checkout__incompatibility-warning"
+		className="express-checkout__notice"
```

Comment:
Just to double-check: we removed the `express-checkout__notice` selector styles and I assume this class name is still needed for the notice to look as expected?